shaders/MicrobeBackground.gdshader

```diff
@@ -22,6 +22,8 @@ uniform float distortionSpeed : hint_range(0.0, 0.01, 0.0001) = 0.001f;
 
 uniform float layerAnimateSpeed : hint_range(0.0, 2.0, 0.1) = 1.0f;
 
+uniform vec2 worldPos;
```

Comment:
Why does world position now need to be passed in? couldn't the math extracting it from the view matrix be just altered?

Reply:
Oh yeah, I kind of suspected after getting to the subviewport. So that part needs to be resolved before talking about this part...